yarn.lock

```diff
@@ -212,7 +212,7 @@
   dependencies:
     "@cucumber/messages" "^13.0.1"
 
-"@cucumber/gherkin@^15.0.0", "@cucumber/gherkin@^15.0.2":
+"@cucumber/gherkin@^15.0.0":
```

Comment:
It's a bit suspicious having both `^15` and `^16` here. I think we should only have `^16`. Let's find out what that's about.

Reply:
We need to make a new release of `gherkin-utils`. The latest release (`2.1.0`) still depends on `@cucumber/gherkin` version  `15`. I'll make a new release of that lib (it won't have a dependency on `@cucumber/gherkin` at all)

Reply:
Okay thanks